web/README.md

```diff
@@ -0,0 +1,418 @@
+
+### DDL
+
+`/ddl` is a GET API which must be used after using conversion APIs (i.e, `/connect` or `/convert`). This API returns the DDL statements of the converted schema.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise DDL statements.
+
+Example
+
+```json
+{
+  "Albums": "--\n-- Spanner schema for source table Albums\n--\nCREATE TABLE Albums (\n    SingerId INT64 NOT NULL, -- From: SingerId smallint(6)\n    AlbumId INT64 NOT NULL,  -- From: AlbumId smallint(6)\n    AlbumTitle STRING(50)    -- From: AlbumTitle varchar(50)\n) PRIMARY KEY (SingerId, AlbumId)"
+}
+```
+
+### Session
+
+(1) `/session` is a GET API which returns the schema conversion state in json format. It also create a file with suffix `.session.json` in the `frontend/` folder.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Session file info with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "FilePath": "/path/to/sessionFile",
+  "FileName": "sessionFile.session.json",
+  "CreatedAt": "2021-01-01T00:00:00.000000+00:00"
+}
+```
+
+(2) `/session/resume` is a POST API which can be used to resume the previous session, it requires the session file which can be generated using the above API. It takes following as request body.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Provide driver and session file path.
+
+Example
+
+```json
+{
+  "driver": "postgres",
+  "path": "/path/to/sessionFile",
+  "fileName": "sessionFile.session.json"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Summary
+
+`/summary` is a GET API which returns table wise summary of the conversion.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise summary of conversion.
+
+Example
+
+```json
+{
+  "Albums": "Note\n1) Some columns will consume more storage in Spanner e.g. for column 'AlbumId', source DB type smallint(6) is mapped to Spanner type int64.\n\n"
+}
+```
+
+### Overview
+
+`/overview` is a GET API which returns the overview of the conversion.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Overall summary of conversion in string format.
+
+### Conversion
+
+`/conversion` is a GET API which returns table wise rate of conversion which is encoded in color values.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise rate of conversion encoded in color values.
+
+Example
+
+```json
+{
+  "Albums": "GREEN",
+  "Singers": "BLUE",
+  "Userinfo": "RED"
+}
+```
+
+### Typemap
+
+(1) `/typemap` is a GET API which returns the source type to list of spanner type map, which can be used for manual type maping from the UI.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Map from source type to list of possible spanner type and issue.
```

Comment:
Suggest rephrasing:

Map from each source type to the list of possible Spanner types and issues.

Reply:
updated statement